pkg/controller/elasticsearch/validation/validations.go

```diff
@@ -30,13 +32,13 @@ const (
 	nodeRolesInOldVersionMsg = "node.roles setting is not available in this version of Elasticsearch"
 	parseStoredVersionErrMsg = "Cannot parse current Elasticsearch version. String format must be {major}.{minor}.{patch}[-{label}]"
 	parseVersionErrMsg       = "Cannot parse Elasticsearch version. String format must be {major}.{minor}.{patch}[-{label}]"
-	pvcImmutableMsg          = "Volume claim templates can only have their storage requests increased, if the storage class allows volume expansion. Any other change is forbidden."
+	pvcImmutableErrMsg       = "volume claim templates can only have their storage requests increased, if the storage class allows volume expansion. Any other change is forbidden"
```

Comment:
`node.roles` is a name of a field so I' think we shouldn't upper case it. The rest is fine, I'd just do:
```suggestion
	pvcImmutableErrMsg       = "Volume claim templates can only have their storage requests increased, if the storage class allows volume expansion. Any other change is forbidden"
```

Reply:
On top of using this as a validation webhook error message, I also use it as a regular k8s error in the reconciliation. Errors are supposed to start with a lowercase so I thought it would be ok to make it lower case also for the validation msg. WDYT?